docs/content/configuration.md

```diff
@@ -242,6 +242,8 @@ multiple services.
 | `services[_].url` | `string` | Yes | Base URL to contact the service with. |
 | `services[_].response_header_timeout_seconds` | `int64` | No (default: 10) | Amount of time to wait for a server's response headers after fully writing the request. This time does not include the time to read the response body. |
 | `services[_].headers` | `object` | No | HTTP headers to include in requests to the service. |
+| `services[_].tls.ca_cert` | `string` | No | The path to the root CA certificate. |
```

Comment:
💭 If these are not required, what's the default?

Reply:
Updated the field description to include defaults.